test/tests/slidedeck.js

```diff
@@ -0,0 +1,50 @@
+/*global sinon,expect,StepTest*/
+/*eslint-disable prefer-arrow-callback*/
+function Init(){
+  this.test("Slidedeck Should Initialize")
```

Comment:
To be consistent, we should either always use SlideDeck/slide-deck or Slidedeck/slidedeck. In classes/folders